src/internal/storage/chunk/uploader.go

```diff
@@ -179,6 +182,8 @@ func (u *Uploader) CopyByReference(meta interface{}, dataRefs []*DataRef) error
 }
 
 func upload(ctx context.Context, client Client, chunkBytes []byte, pointsTo []ID, noUpload bool) (*DataRef, error) {
+	ctx = pctx.Child(ctx, "upload", pctx.WithCounter("bytes", 0))
```

Comment:
It seems like this counter should be set in the subcontext above. Also, the names don't match.